proposals/lip-0048.md

```diff
@@ -204,12 +205,27 @@ def getFeeTokenID() -> TokenID:
     return TOKEN_ID_FEE
 ```
 
+### Genesis Block Processing
+
+#### Genesis State Initialization
+
+```python
+def initGenesisState(b: Block) -> None:
+    if ADDRESS_FEE_POOL is not None:
+        Token.initializeUserAccount(ADDRESS_FEE_POOL, TOKEN_ID_FEE)
+```
+
 ### Block Processing
 
 #### Verify Transaction
 
 ```python
 def verify(trs: Transaction) -> None:
+    b = block including trs
+    h = b.header.height
+    if h < MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE:
+	    # Set MIN_FEE_PER_BYTE = 0 for the first MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE blocks after the genesis block has been generated.
```

Comment:
"after the genesis block has been generated." this seems confusing to me since genesis block is also included for `ZERO_FEE_PER_BYTE`.

Reply:
I agree. Maybe I could change it to simply say `# Set MIN_FEE_PER_BYTE = 0 for the first MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE blocks.`?